packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -155,40 +130,40 @@
 .label {
   display: block;
   position: relative;
-  margin-block-end: $spacing-xs;
-  padding-inline-start: 1em + $spacing-s-em;
-  font-size: $size-body-base;
-  font-weight: 400;
+  margin-block-end: var(--MarginBottom);
+  padding-inline-start: calc(var(--BoxSize) + var(--PaddingLabelLeft));
+  font-size: var(--FontSize);
+  font-weight: var(--FontWeight);
   text-align: start;
 
   .box,
   .indicator {
-    @include border-radius($base-border-radius);
+    @include border-radius(var(--BorderRadius));
 
     display: block;
     position: absolute;
     inset-block-start: 50%;
     inset-inline-start: 0;
     transform: translateY(-50%);
     transition-property: border-color, color, opacity, box-shadow;
-    transition-duration: $base-duration;
-    transition-timing-function: $base-timing;
+    transition-duration: var(--TransitionDuration);
+    transition-timing-function: var(--TransitionTiming);
   }
 
   /* Box */
 
   .box {
-    width: 1em;
-    height: 1em;
-    border: 2px solid $border-color-ui;
+    width: var(--BoxSize);
+    height: var(--BoxSize);
+    border: var(--BorderSize) solid var(--ColorBorder);
```

Comment:
`border.style.base` is available if you'd like to use it here

Reply:
👍